src/views/components/header.html

```diff
@@ -2,7 +2,7 @@
 	<nav class="navbar navbar-default {{!actualPath || withFullWhiteLogoRoutes.includes(actualPath) ? 'navbar-filled' : ''}}">
 		<div class="container-fluid container-mw">
 			<div class="navbar-header">
-				<a href="/">
```

Comment:
What issue is this related to?

Reply:
According to the Lighthouse we should not use a tags without href to increase SEO performance. I have another task to fix all SEO issues we have. Just replaced this tag right now since after fix it has no href anymore.

Reply:
But it has a href. I meant overall why the change of moving it to JS.

Reply:
Because when it is a regular a-tag with href to the root it doesn't work in the way asked for in the task 467, it doesn't drop query params
![22](https://user-images.githubusercontent.com/13244319/213778074-d41a613d-3796-4569-b1c2-fab04b15a7ec.png)